ast/builtins.go

```diff
@@ -2327,6 +2328,37 @@ with all others being treated as intermediates.`,
 	),
 }
 
+var CryptoX509ParseAndVerifyCertificatesWithOptions = &Builtin{
+	Name: "crypto.x509.parse_and_verify_certificates_with_options",
+	Description: `Returns one or more certificates from the given string containing PEM
+or base64 encoded DER certificates after verifying the supplied certificates form a complete
+certificate chain back to a trusted root. Second argument is a option object to validate the 
```

Comment:
This line looks to be incomplete? the note about the options seems to be cut off.

Reply:
Hi,
its giving me following error on last step. I didn't have `hugo` so I installed it by `sudo apt  install hugo`

~/opa/docs/website$ **hugo server --contentDir=generated/**
```
Building sites … ERROR 2024/02/21 11:55:13 render of "page" failed: "/home/yogi/opa/docs/website/layouts/docs/ecosystem-single.html.html:20:17": execute of template failed: template: docs/ecosystem-single.html.html:20:17: executing "content" at <path>: can't evaluate field BaseName in type interface {}
ERROR 2024/02/21 11:55:13 render of "page" failed: "/home/yogi/opa/docs/website/layouts/docs/ecosystem-single.html.html:20:17": execute of template failed: template: docs/ecosystem-single.html.html:20:17: executing "content" at <path>: can't evaluate field BaseName in type interface {}
ERROR 2024/02/21 11:55:13 render of "page" failed: "/home/yogi/opa/docs/website/layouts/docs/ecosystem-single.html.html:20:17": execute of template failed: template: docs/ecosystem-single.html.html:20:17: executing "content" at <path>: can't evaluate field BaseName in type interface {}
ERROR 2024/02/21 11:55:13 render of "page" failed: "/home/yogi/opa/docs/website/layouts/docs/ecosystem-single.html.html:20:17": execute of template failed: template: docs/ecosystem-single.html.html:20:17: executing "content" at <path>: can't evaluate field BaseName in type interface {}
Built in 1659 ms
Error: Error building site: failed to render pages: render of "page" failed: "/home/yogi/opa/docs/website/layouts/docs/ecosystem-single.html.html:20:17": execute of template failed: template: docs/ecosystem-single.html.html:20:17: executing "content" at <path>: can't evaluate field BaseName in type interface {}
```

Reply:
Hmm, I am not sure what would be causing that... 🤔 

how about here: https://deploy-preview-6583--openpolicyagent.netlify.app/docs/edge/policy-reference/#builtin-crypto-cryptox509parse_and_verify_certificates_with_options